Reference/Searching/Examine/Indexing/index.md

```diff
@@ -6,6 +6,185 @@ versionFrom: 8.0.0
 
 Examine has changed quite a bit in Umbraco 8 (and by "a bit" we really mean a lot). In Umbraco 7 everything was configured in the two Examine config files - in Umbraco 8 everything happens through C#.
 
+## Customizing the built in indexes
+
+You can modify the built in indexes in several ways:
+
+* Events - giving you control over exactly what data goes into them and how the fields are configured
+* Changing the field value types to change how values are stored in the index
+* Changing the IValueSetValidator to change what goes into the index
+* Take control of the entire index creation pipeline to completely change the implementation
+
+### Changing field value types
+
+By default Examine will store values into the Lucene index as "Full Text", meaning it will be indexed and analyzed for a textual search but if a value that you are storing in a field is numerical or a date/time or another specific value type then you might want to change how this is stored in the index to take advantage of some features such as numerical range or date features, etc...
+
+There is some documentation about this in the [Examine documentation](https://shazwazza.github.io/Examine/configuration).
+
+The easiest way to modify how a field is configured is using a custom component, for example:
+
+```c#
+// register the component
+public class CustomizeIndexComposer : ComponentComposer<CustomizeIndexComponent> { }
+
+public class CustomizeIndexComponent : IComponent
+{
+    private readonly IExamineManager _examineManager;
+
+    public CustomizeIndexComponent(IExamineManager examineManager)
+    {
+        _examineManager = examineManager;
+    }
+
+    public void Initialize()
+    {
+        // get the external index
+        if (!_examineManager.TryGetIndex(UmbracoIndexes.ExternalIndexName, out IIndex index))
+            return;
+
+        // add a custom field type
+        index.FieldDefinitionCollection.TryAdd(new FieldDefinition("price", FieldDefinitionTypes.Double));
+
+        // modify an existing field type (not recommended)
+        index.FieldDefinitionCollection.AddOrUpdate(new FieldDefinition("parentID", FieldDefinitionTypes.FullText));
+    }
+
+    public void Terminate()
+    {
+    }
+}
+```
+
+## Changing IValueSetValidator
+
+An `IValueSetValidator` is responsible for validating a `ValueSet` to see if it should be included in the index and/or filtering the data in the `ValueSet`. For example, by default the validation process for the ExternalIndex checks if a `ValueSet` has a category type of either "media" or "content" (not member), if a `ValueSet` was passed to the ExternalIndex that did not pass this requirement it would just be ignored.
```

Comment:
This sentence was a bit tough to split up. Does the whole section still make sense? 🤔

```suggestion
An `IValueSetValidator` is responsible for validating a `ValueSet` to see if it should be included in the index and/or filtering the data in the `ValueSet`. For example, by default the validation process for the ExternalIndex checks if a `ValueSet` has a category type of either "media" or "content" (not member). If a `ValueSet` was passed to the ExternalIndex and it did not pass this requirement it would be ignored.
```